### PR TITLE
updated pending reviews page

### PR DESCRIPTION
**Pull request by Jaredh55**

### CODE REVIEW

File: app/controllers/employees_controller.rb [hunk 1]
```diff
@@ -4,11 +4,7 @@ def index
      @employees = Employee.all
      render 'index.json.jbuilder'
   end
-  def manager_index
```
Comment (afzalmasra125) [inline app/controllers/employees_controller.rb]: Why did we remove? currently, this allows us to get all manager in the system

File: app/controllers/employees_controller.rb [hunk 2]
```diff
@@ -27,6 +23,7 @@ def create
     end
   def show
     @employee = Employee.find(params[:id])
+    p @employee.reviews
```
Comment (afzalmasra125) [inline app/controllers/employees_controller.rb]: p is for debugging purposes, Lets clean that

File: app/controllers/reviews_controller.rb [hunk 3]
```diff
@@ -21,7 +23,7 @@ def create
     if reviews.save
       render json: reviews.as_json
     else
-      reviews.errors
+      p reviews.errors
```
Comment (afzalmasra125) [inline app/controllers/reviews_controller.rb]: p is for debugging purposes, Lets clean that

File: public/index.html [hunk 4]
```diff
@@ -224,6 +224,44 @@ <h2>Review</h2>
...
     <div class="container">
       <h1>Pending Reviews</h1>
+        <h2>Manager</h2>
+
+        <select v-model="selectedManager">
+          <option value="">Please Select</option>
+          <option
+            v-for="employee in employees"
+            :value="employee.id"
+            v-if="employee.manager_status == true"
```
Comment (afzalmasra125) [inline public/index.html]: API that is removed - manager index is taking care of this logic
It can be cleaned up later

**Comment by Jaredh55** [comment]
@afzalmasra125 we made the changes, can we merge?

**Pull request merged**